tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -192,7 +176,6 @@ public void processOsm(SourceFeature sf, FeatureCollector features) {
         switch (minAdminLevel.getAsInt()) {
           case 2 -> {
             kind = "country";
-            kindDetail = "2";
```

Comment:
Be wary of removing `kindDetail` here as `kind` alone doesn't capture some of the per country variance –– right now Protomaps v3 and v4 would push _more_ boundaries features from OSM into tiles than Tilezen v1.9 schema, and the `kindDetail` would provide a way to filter those out in the style. The better fix would be to be more selective at tile cut, of course.